sdk/eventgrid/azure-eventgrid/azure/eventgrid/_generated/aio/operations/_event_grid_publisher_client_operations.py

```diff
@@ -22,7 +22,7 @@ class EventGridPublisherClientOperationsMixin:
     async def publish_events(
         self,
         topic_hostname: str,
-        events: List["models.EventGridEvent"],
+        events: List["_models.EventGridEvent"],
```

Comment:
random question: how would this be rendered on doc/type hint?

wondering if users would get confused that how they create `EventGridEvent` from `_models` module.
(ignore me if this could be rendered corrected by python/sphinx)

Reply:
I'm not much concerned about this - this is an autogenerated method from swagger which is not used anywhere in our code. (we just need this in the swagger for the sake of other languages)